src/components/Envelope.vue

```diff
@@ -400,7 +400,7 @@ export default {
 		},
 		addresses() {
 			// Show recipients' label/address in a sent mailbox
-			if (this.mailbox.specialRole === 'sent') {
+			if (this.mailbox.specialRole === 'sent' || this.mailbox.sentMailboxId === this.account.databaseId) {
```

Comment:
```suggestion
			if (this.mailbox.specialRole === 'sent' || this.account.sentMailboxId === this.mailbox.databaseId) {
```

Reply:
You're right, thank you for catching it :)

Reply:
You are welcome